### PR TITLE
weight proof 

### PR DESCRIPTION
**Pull request by almogdepaz**

### CODE REVIEW

File: tests/core/consensus/test_weight_proof.py [hunk 1]
```diff
@@ -306,11 +308,11 @@ async def test_weight_proof_extend_multiple_ses(self, default_1000_blocks):
     async def test_weight_proof_from_database(self):
         connection = await aiosqlite.connect("path to db")
         block_store: BlockStore = await BlockStore.create(connection)
-        sub_blocks, peak = await block_store.get_sub_block_records()
-        headers = await block_store.get_header_blocks_in_range(0, 100225)
```
Comment (mariano54) [inline tests/core/consensus/test_weight_proof.py:310]: Why are we reducing this? It seems like this would make the test catch less issues

**Review by mariano54 — COMMENTED** [review]
What is the performance like for validating one of these? We should make sure it's not too slow before merging it into testnet

File: tests/core/consensus/test_weight_proof.py (hunk 1 above)
Comment (almogdepaz) [reply tests/core/consensus/test_weight_proof.py:310]: this is just a harness for debugging, doesn't run with ci, works with the testnet database

**Comment by almogdepaz** [comment]
closing and opening new pr to avoid merge

**Pull request closed without merging**